### ISSUE TITLE
apostrophes are bad for api calls

### ISSUE THREAD

**Issue by mattkiefer** [mattkiefer/foiamail#20]
So far, only seems to affect attachment collection for a small number of agencies, e.g.

```python mgr.py --atts

skipping Troy FPD
Governor's State University failed <HttpError 400 when requesting https://www.googleapis.com/drive/v3/files?q=name%3D%27Governor%27s+State+University%27&alt=json returned "Invalid Value">
skipping DeWitt County
```

TODO: fix

**Comment by mattkiefer** [comment]
TODO: see if you can str.replace('\'','') agency name just for Drive API calls in atts.gm.roll_thru()[18:24]

**Comment by mattkiefer** [comment]
tried to fix with c06602785f9db9092ae6ed2ea86da6871fb87776 but still broken

**Comment by mattkiefer** [comment]
fixed in f1c5106 and verified
